frontend/pendingTranslations.ftl

```diff
@@ -7,6 +7,7 @@ nav-phone-dashboard = Phone Masks
 nav-duo-description = Switch dashboards
 nav-duo-email-mask-alt = Email masks
 nav-duo-phone-mask-alt = Phone masks
+nav-footer-stock-photo-legal = Model photographs on this page are sourced from iStock, © iStockPhoto.com
```

Comment:
Apparently there were a lot of strings with trailing whitespace in this file, which my editor automatically removed - but this is the only new string.